packages/core/src/accessanalyzer/vue/iamPolicyChecks.ts

```diff
@@ -695,6 +745,30 @@ function getResultCssColor(resultType: PolicyChecksResult): string {
     }
 }
 
+function isCloudFormationTemplate(document: string): boolean {
+    if (document.endsWith('.yaml') || document.endsWith('.yml') || document.endsWith('.json')) {
+        return true
+    } else {
+        return false
+    }
```

Comment:
nit: Applicable for each function
```suggestion
   return document.endsWith('.yaml') || document.endsWith('.yml') || document.endsWith('.json'))
```

or even better
```
const cfnFileTypes = ['.yaml', '.yml', '.json']
const terraformPlanFileTypes = ['.json']
...
```
```suggestion
return cfnFileTypes.some(t => document.endsWith(t))
```